rundeck/resource_job.go

```diff
@@ -321,13 +321,37 @@ func resourceRundeckJob() *schema.Resource {
 							Optional: true,
 							Elem:     resourceRundeckJobPluginResource(),
 						},
+						"error_handler": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem:     resourceRundeckJobCommandResource(),
+						},
 					},
 				},
 			},
 		},
 	}
 }
 
+func resourceRundeckJobCommandResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"keep_going_on_success": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Type is not actually a required property for the error handler. In the case of a simple exec command step as the handler it looks like this in the xml export:
```
<errorhandler keepgoingOnSuccess='true'>
    <exec>true</exec>
</errorhandler>
```